CONTRIBUTING.md

```diff
@@ -43,6 +43,24 @@ GitHub provides additional document on [forking a repository](https://help.githu
 ## Finding contributions to work on
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/awslabs/aws-lambda-cpp-runtime/labels/help%20wanted) issues is a great place to start. 
 
+## Running the Integration Tests Locally
+
+The integration testing for the project creates, invokes, and deletes, Lambda functions. 
+These tests typically run  AWS CodeBuild, but may also be executed locally
```

Comment:
```suggestion
These tests typically run on AWS CodeBuild, but may also be executed locally
```

So this literally runs CodeBuild locally using CodeBuild's docker image, right?

Reply:
yup!